src/resources/Enums.ts

```diff
@@ -5,7 +5,7 @@ export enum AccessLevel {
     NONE = 'NONE',
 }
 
-export enum PrivilegeholderType {
+export enum PrivilegeHolderType {
     API_KEY = 'API_KEY',
     GROUP = 'GROUP',
 }
```

Comment:
If this is exported publicly and it's old enough that there are significant risks that some people are using it, I would recommend also exporting the old name, until we do a major version bump:
```suggestion
export enum PrivilegeHolderType {
    API_KEY = 'API_KEY',
    GROUP = 'GROUP',
}

/**
 * @deprecated use `PrivilegeHolderType` instead.
 */
export type PrivilegeholderType = PrivilegeHolderType;
```

Reply:
Yes good idea. The changes to the `AccessModel` will still be breaking, but at least the `PrivilegeHolderType ` enum itself won't be. In case someone would use this alone without the **AccessModel**.